buildSrc/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+  id "org.gradle.kotlin.kotlin-dsl" version "4.3.0"
+}
+
+repositories {
+  gradlePluginPortal()
+  google()
+}
+
+dependencies {
+  // Needed to fix issue with android kotlin project
+  implementation("com.android.tools.build:gradle:8.2.2")
+  implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.22")
```

Comment:
No `libs.versions.toml` in this repo? You can point at the enclosing project with https://github.com/cashapp/redwood/blob/trunk/build-support/settings.gradle#L6-L12